rollup.config.js

```diff
@@ -0,0 +1,30 @@
+import babel from "rollup-plugin-babel";
+import uglify from "rollup-plugin-uglify";
+
+const name = process.env.NODE_ENV === "production"
+  ? "freactal.umd.min.js"
+  : "freactal.umd.js";
+
+const config = {
+  input: "./src/index.js",
+  output: {
+    file: `./umd/${name}`,
+    directory: "umd",
+    format: "umd"
+  },
+  name: "Freactal",
+  external: ["react", "prop-types"],
+  globals: {
+    "react": "React",
+    "prop-types": "PropTypes"
```

Comment:
While we're at it -- should we move `prop-types` from `dependencies` to `peerDependencies`? That's typically how I see most projects doing it these days...

/cc @divmain

Reply:
Turns out documentation says the other way (actual dep now...) I'll switch it back.